src/main.ts

```diff
@@ -180,6 +204,18 @@ export default class LinterPlugin extends Plugin {
       new Notice('Linted all files');
     }
 
+    async runLinterAllFileInFolder(folder: TFolder) {
+      console.log('Linting folder ' + folder.name);
+      let lintedFiles = 0;
+      await Promise.all(this.app.vault.getMarkdownFiles().map(async (file) => {
+        if (file.path.startsWith(folder.path) && !this.shouldIgnoreFile(file)) {
```

Comment:
when I lint the folder named `Test` it also lints files starting with `Test` like Testing.md which it shouldnt

Reply:
This should be taken care of with the latest commit.